setup.cfg

```diff
@@ -49,8 +49,8 @@ dev =
     pytest
 sync =
     click
-    gdcdictionary
-    gdcdatamodel2
+    gdcdictionary~=3.0.0
```

Comment:
Why pin? Were you getting 3.1.0 artifacts in the requirements.txt?

Reply:
I'm pinning to keep this PR from being about the actual upgrade of the mappings. That will be part of work to come.

Reply:
Did a quick search and I do not think we need gdcdictionary at all

Reply:
Not sure where you were searching but the sync process uses gdcdictionary to load the descriptions for the properties [here](https://github.com/NCI-GDC/gdc-models/blob/0b000e9ed2532a2d3b41f2c2543f1ee7ce3d544a/src/gdcmodels/sync/graph/common.py#L102-L119).